src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/cdk/cdk_stacks/adf_standard_pipelines.py

```diff
@@ -0,0 +1,260 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
On the filename, at the moment, it is called `adf_standard_pipelines.py` whereas you called the Pipelines Type "Default", maybe good to relate the two. Plus the name is plural: `pipelines`, whereas it is used to generate a single pipeline. What do you think about calling it `adf_default_pipeline.py` instead?

Reply:
Makes sense. 👍